### ISSUE TITLE
Master Issue to track Preprocessing Layer Support for BBox and Segmentation Masks

### ISSUE THREAD

**Issue by ariG23498** [keras-team/keras-cv#1992]
Segmentation Masks:
- [x] AugMix: https://github.com/keras-team/keras-cv/pull/1988
- [x] FourierMix: https://github.com/keras-team/keras-cv/pull/1991
- [ ] RandAugment
- [x] RandomCrop: https://github.com/keras-team/keras-cv/pull/2027
- [x] RandomCutout: https://github.com/keras-team/keras-cv/pull/2004
- [x] RandomShear: https://github.com/keras-team/keras-cv/pull/2021
- [x] RandomTranslation: https://github.com/keras-team/keras-cv/pull/2024
- [x] RandomZoom - https://github.com/keras-team/keras-cv/pull/2010
- [ ] RandomAspectRatio

BBox:
- [ ] CutMix
- [ ] FourierMix
- [x] RandAugment: https://github.com/keras-team/keras-cv/issues/728
- [x] RandomCutout: https://github.com/keras-team/keras-cv/issues/730
- [ ] RandomZoom - @ariG23498 

The list directly comes from [this README](https://github.com/keras-team/keras-cv/tree/master/keras_cv/layers/preprocessing).

CC: @ianstenbit

**Comment by innat** [comment]
Partially similar. 
https://github.com/keras-team/keras-cv/discussions/93

**Comment by ariG23498** [comment]
Hi @cosmo3769 -- I have noticed you taking up on a lot of segmentation map layers 🤗 . Thanks for that. Could you first claim the layers in this issue? -- That would help us track the progress better.

**Comment by cosmo3769** [comment]
@ariG23498 Surely. I would tag this issue first before opening a new PR.

**Comment by cosmo3769** [comment]
Hi @ariG23498, I am working on RandomShear layer.

**Comment by cosmo3769** [comment]
@ariG23498 Could you please add RandomAspectRatio in the 'Segmentation Mask' section in the description above. It's missing here.

**Comment by IMvision12** [comment]
@ariG23498 Taking up RandomTranslation and RandomCrop

**Comment by Zekrom-7780** [comment]
@ariG23498 can I take Cutmix and FourierMix in BBox?

**Comment by ariG23498** [comment]
> @ariG23498 can I take CutMix and FourierMix in BBox?

Hey @Zekrom-7780 please feel free to take them up! As soon as you have made a PR link this issue and I will update the description accordingly. Thanks!

**Comment by cosmo3769** [comment]
> * [x]  RandomCrop: [[RandomTranslation] Supporting Segmentation Masks #2024](https://github.com/keras-team/keras-cv/pull/2024)

Hi @ariG23498, the correct link is [[RandomCrop] Supporting Segmentation Masks #2027](https://github.com/keras-team/keras-cv/pull/2027). Please update. Thank you.

**Comment by sachinprasadhs** [comment]
Thanks for reporting the issue! We have consolidated the development of KerasCV into the new KerasHub package, which supports image, text, and multi-modal models. Please read https://github.com/keras-team/keras-hub/issues/1831. KerasHub will support all the core functionality of KerasCV.

KerasHub can be installed with !pip install -U keras-hub. Documentation and guides are available at [keras.io/keras_hub](https://keras.io/keras_hub/).

With our focus shifted to KerasHub, we are not planning any further development or releases in KerasCV. If you encounter a KerasCV feature that is missing from KerasHub, or would like to propose an addition to the library, please file [an issue with KerasHub](https://github.com/keras-team/keras-hub/issues).
